scss/modules/_datatables.scss

```diff
@@ -115,10 +115,15 @@
     background-color: $inverse;
     border-bottom: 1px solid $neutral;
 
+    &.row-color-contrast,
     &:nth-child(even) {
       background-color: rgba($gray-lightest, .5);
     }
 
+    &.row-color-normal {
```

Comment:
This is just for my own understanding: if line 115 sets the whole table background to `$inverse`, why do you need to set a separate `normal` row color to the same color?